Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/keysubmission/AnalyticsKeySubmissionCollector.kt

```diff
@@ -30,6 +30,7 @@ class AnalyticsKeySubmissionCollector @Inject constructor(
 
     fun reportPositiveTestResultReceived(type: CoronaTest.Type) {
         if (disabled) return
+        if (type.storage.testResultReceivedAt.value > 0) return
```

Comment:
Suggestion: Maybe do something like the code below to make it a bit more understandable? Or add a comment?

```kotlin
val alreadySet = type.storage.testResultReceivedAt.value != -1
if(!alreadySet){
    type.storage.testResultReceivedAt.update { timeStamper.nowUTC.millis }
}

```